processing/data_collection/gazette/pipelines.py

```diff
@@ -1,5 +1,5 @@
+import re
```

Comment:
these changes look like unrelated, or am I missing something here?

Reply:
Yes, that was left over from the middleware that was removed.
I reverted the file.